compiler/rustc_middle/src/ty/consts.rs

```diff
@@ -242,7 +241,7 @@ impl<'tcx> Const<'tcx> {
         if let Some(val) = self.val.try_eval(tcx, param_env) {
             match val {
                 Ok(val) => Const::from_value(tcx, val, self.ty),
-                Err(ErrorReported) => tcx.const_error(self.ty),
+                Err(/* ErrorReported */ _) => tcx.const_error(self.ty),
```

Comment:
Does `Err(ErrorReported {..})` work?

Reply:
Apparently it does... but `ErrorReported(_)` does not? I'm not sure how to think about that...